boards/opentitan/README.md

```diff
@@ -20,19 +20,16 @@ Programming
 -----------
 
 Tock on OpenTitan requires
-lowRISC/opentitan@199d45626f8a7ae2aef5d9ff73793bf9a4233711 or newer. In
-general it is recommended that users start with the latest OpenTitan bitstream
-and if that results in issues try the one mentioned above.
+lowRISC/opentitan@199d45626f8a7ae2aef5d9ff73793bf9a4233711. In
+general it is recommended that users start with the specified commit as newer
+versions have not been tested.
 
-For more information you can follow the
-[OpenTitan development flow](https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html#testing-the-demo-design)
-to flash the image.
-
-First setup the development board using the steps here:
-https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html.
-You need to make sure the boot ROM is working and that your machine can
-communicate with the OpenTitan ROM. You will need to use the `PROG` USB
-port on the board for this.
+In general the OpenTitan project has
```

Comment:
Please cut this link -- better to just point people at what the right thing to do is than say "Here's the documentation, actually that's the wrong documentation instead look at this other documentation."